src/simpleTrianglePatterns/simpleTrianglePattern2.java

```diff
@@ -7,19 +7,19 @@ public static void main(String[] args){
         Scanner userInputScanner = new Scanner(System.in);
         System.out.print("Enter a number:");
         try{
-            int number = userInputScanner.nextInt();
-            for(int i = 1; i <= number; i++){
-                for(int j = i; j <= number; j++){
+            int UserInputNumber = userInputScanner.nextInt();
+            for(int i = 1; i <= UserInputNumber; i++){
+                for(int j = i; j <= UserInputNumber; j++){
                     System.out.print(" ");
                 }
                 for(int j = 1;j <= i; j++){
                     System.out.print("*");
                 }
                 System.out.println();
             }
-        } catch (RuntimeException ex2){
```

Comment:
Lets check how exceptions works, why is it a bad thing that you uses Exception to write out the wrong exception? also, runtime exception not needed here. Here will be only 1 very specific exception which is thrown when the input is not a number!